.github/workflows/rebuild-qgis-25.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
           --tag=camptocamp/geomapfish-qgisserver \
           docker/qgisserver;
           docker tag camptocamp/geomapfish-qgisserver \
-          camptocamp/geomapfish-qgisserver:gmf${MAJOR_VERSION}-qgis${{ matrix.version }}
+          camptocamp/geomapfish-qgisserver:gmflatest-qgis${{ matrix.version }}
```

Comment:
I do not understand why you tag gmflatest in rebuild-qgis-25.yaml

Reply:
I do the same in the build, here:
https://github.com/camptocamp/c2cgeoportal/blob/2.5/.github/workflows/qgis.yaml#L58
It is because c2cciutils want to retag from latest to 2.5, see:
https://github.com/camptocamp/c2cgeoportal/runs/1561440802?check_suite_focus=true#step:10:23

Reply:
Ok, you have another problem : Error: Command '['docker', 'tag', 'camptocamp/geomapfish-qgisserver:gmflatest-qgis3.4', 'ghcr.io/camptocamp/geomapfish-qgisserver:gmf3.4-qgis3.4']' returned non-zero exit status 1.
=> gmf3.4-qgis3.4